pandas/tests/groupby/test_categorical.py

```diff
@@ -67,6 +68,12 @@ def setUp(self):
              'E': np.random.randn(11),
              'F': np.random.randn(11)})
 
+    def test_level_groupby_get_group(self):
+        # gh15155
+        testGroupBy = self.mframe.groupby(level=["first"])
+        assert_numpy_array_equal(testGroupBy.get_group("foo").values,
```

Comment:
I can't use `assert_frame_equal(a,b)` since the two have different structures, at least not as it is:
```
# testGroupBy.get_group("foo")
                     A         B         C
first second
foo   one    -0.529506 -1.282295  0.521247
      two     0.982640  0.428228  1.031026
      three  -0.753013  0.087009  1.082119

# self.mframe.loc["foo"]
               A         B         C
second
one    -0.529506 -1.282295  0.521247
two     0.982640  0.428228  1.031026
three  -0.753013  0.087009  1.082119
```
I agree that testing frames would be nicer, but how?